interface/globals.php

```diff
@@ -565,25 +565,6 @@ function GetCallingScriptName()
 //    -if you don't want it translated, then strip the xl function away
 $tback = xl('(Back)');
 
-$version = (new VersionService())->fetch();
-
-if (!empty($version)) {
-    //Version tag
-    $patch_appending = "";
-    //Collected below function call to a variable, since unable to directly include
-    // function calls within empty() in php versions < 5.5 .
-    $version_getrealpatch = $version['v_realpatch'];
-    if (($version['v_realpatch'] != '0') && (!(empty($version_getrealpatch)))) {
-        $patch_appending = " (" . $version['v_realpatch'] . ")";
-    }
-
-    $openemr_version = $version['v_major'] . "." . $version['v_minor'] . "." . $version['v_patch'];
-    $openemr_version .= $version['v_tag'] . $patch_appending;
-} else {
-    $openemr_version = xl('Unknown version');
-}
-$GLOBALS['openemr_version'] = $openemr_version;
-
 $srcdir = $GLOBALS['srcdir'];
```

Comment:
$GLOBALS['openemr_version'] is used in portal footer to keep up with core versioning.
Used in `sql_patch.php`,  `portal/patient/templates/_Footer.tpl.php`, `src/Services/ProductRegistrationService.php`

Reply:
recm just grabbing from version table and format there in globals.